### PR TITLE
[CI] Input parameter to ignore test failures

### PR DESCRIPTION
**Pull request by pbchekin**
It is possible to run "Build and test" workflow manually and set `ignore_errors` to true to make sure the workflow executes all test suites and report failures. Using a common code to run tutorials in CI and in test-triton.sh.

### CODE REVIEW

**Review by gshimansky — APPROVED** [review]

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -11,6 +12,14 @@ on:
...
         type: boolean
         default: false
+      ignore_errors:
+        description: Ignore test errors
+        type: boolean
+        default: false
+      run_name:
+        description: Custom run name
+        type: string
+        default: "Build and test"
```
Comment (gshimansky) [inline .github/workflows/build-test.yml:22]: It doesn't seem to be used anywhere in the workflow

Comment (pbchekin) [reply .github/workflows/build-test.yml:22]: It is used in line 2 to set a custom run name (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#run-name). I am planning to use the run name by triggering this workflow externally, passing parameters, including the run name, for example, "Build and test on PVC with LTS".

File: .github/workflows/build-test.yml [hunk 2]
```diff
@@ -31,6 +40,10 @@ jobs:
       - spr
       - cpu
     steps:
+      - name: Print inputs
```
Comment (leshikus) [inline .github/workflows/build-test.yml:43]: I like this, because github sometimes puts `null` for inputs

**Review by leshikus — APPROVED** [review]
LGTM

**Pull request merged**